teamcity: failed tests on master: acceptance/TestNodeRestart

The following tests appear to have failed:

[#37122](https://teamcity.cockroachdb.com/viewLog.html?buildId=37122):

```
--- FAIL: acceptance/TestNodeRestart (12.883s)
	chaos_test.go:270: pq: transaction commit result is ambiguous

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/acceptance/chaos_test.go
+++ b/pkg/acceptance/chaos_test.go
@@ -147,7 +147,7 @@ func verifyAccounts(t *testing.T, client *testClient) {
 		client.RLock()
 		defer client.RUnlock()
 		err := client.db.QueryRow("SELECT SUM(balance) FROM bank.accounts").Scan(&sum)
-		if err != nil && !testutils.IsSQLRetryError(err) {
+		if err != nil && !testutils.IsSQLRetryableError(err) {
 			t.Fatal(err)
 		}
 		return err
@@ -164,7 +164,7 @@ func transferMoneyLoop(idx int, state *testState, numAccounts, maxTransfer int) 
 	for !state.done() {
 		if err := transferMoney(client, numAccounts, maxTransfer); err != nil {
 			// Ignore some errors.
-			if !testutils.IsSQLRetryError(err) {
+			if !testutils.IsSQLRetryableError(err) {
 				// Report the err and terminate.
 				state.errChan <- err
 				break

--- a/pkg/acceptance/dynamic_client.go
+++ b/pkg/acceptance/dynamic_client.go
@@ -78,7 +78,7 @@ func (dc *dynamicClient) exec(query string, args ...interface{}) (gosql.Result, 
 		}
 		if result, err := client.Exec(
 			query, args...,
-		); err == nil || !testutils.IsSQLRetryError(err) {
+		); err == nil || !testutils.IsSQLRetryableError(err) {
 			return result, err
 		}
 	}
@@ -95,7 +95,7 @@ func (dc *dynamicClient) queryRowScan(query string, queryArgs, destArgs []interf
 		}
 		if err := client.QueryRow(
 			query, queryArgs...,
-		).Scan(destArgs...); err == nil || !testutils.IsSQLRetryError(err) {
+		).Scan(destArgs...); err == nil || !testutils.IsSQLRetryableError(err) {
 			return err
 		}
 	}

--- a/pkg/testutils/error.go
+++ b/pkg/testutils/error.go
@@ -51,11 +51,13 @@ func IsPError(pErr *roachpb.Error, re string) bool {
 	return matched
 }
 
-// IsSQLRetryError returns true if err is retryable. This is true for errors
-// that show a connection issue or an issue with the node itself. This can
-// occur when a node is restarting or is unstable in some other way.
-func IsSQLRetryError(err error) bool {
-	return IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF)")
+// IsSQLRetryableError returns true if err is retryable. This is true
+// for errors that show a connection issue or an issue with the node
+// itself. This can occur when a node is restarting or is unstable in
+// some other way. Note that retryable errors may occur event in cases
+// where the SQL execution ran to completion.
+func IsSQLRetryableError(err error) bool {
+	return IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF|commit result is ambiguous)")
 }
 
 // Caller returns filename and line number info for the specified stack

